### ISSUE TITLE
Broken frame-element-select

### ISSUE THREAD

**Issue by hendursaga** [atlas-engineer/nyxt#2293]
After opening https://azlyrics.com/ with latest Nyxt, running `toggle-debug-on-error` and then `select-frame-new-buffer`, the frame source selection fails entirely, turning that entire buffer (as in, the prompt buffer is still visible) completely black and freezing everything.

```
<WARN> [17:22:10] Warning: JavaScript error: GError: Domain: "WebKitJavascriptError", Code: 699, Message: https://www.azlyrics.com/:45:49: ReferenceError: Can't find variable: selection
```

I assume it has something to do with https://github.com/atlas-engineer/nyxt/blob/master/source/mode/document.lisp#L576 ?? I'm no ParenScript wizard (yet). I do see that [#1651](https://github.com/atlas-engineer/nyxt/issues/1651#issuecomment-1136262106) has new activity..

While you're at it, https://github.com/atlas-engineer/nyxt/blob/master/source/mode/document.lisp#L643 seems like duplicated code, as we already have that PS macro..

**Comment by hendursaga** [comment]
An update: on 855f8ba374c513c8f795a0d66afe319c983a48f0, I can now run `select-frame-new-buffer` and it opens the cookie consent frame, but now I'm getting error messages with no real message to them, like

`<WARN> [11:39:12] Warning: JavaScript error: GError: Domain: "WebKitJavascriptError", Code: 699, Message:`

Not sure how to proceed to debugging this.

**Comment by Ambrevar** [comment]
@aartaka @aadcg What's the status on this?

**Comment by Ambrevar** [comment]
@hendursaga Was it working properly some time in the past?

**Comment by aartaka** [comment]
> @aartaka @aadcg What's the status on this?

I guess it should still be broken, because we haven't even touched it while refactoring `hint-mode`.

**Comment by aadcg** [comment]
I have not used this feature before, but let me attempt to fix it.

**Comment by hendursaga** [comment]
@aadcg I actually haven't used this feature before I found out it was broken - I was looking for bugs to squash! :bug: 

Now it doesn't show any frames to select, but the rest of the buffer does go black, and the error "message" is the same, although I've noticed I could `C-g` out of it and things return to normal.

Testing with something like https://iframe-html.com/ still doesn't work - it won't show any frames to select.
